docker/build.sh

```diff
@@ -23,3 +23,4 @@ cd $ROOT_DIR/docker
 
 mvn -f ../dashboard/pom.xml package -Pdocker
 mvn package -Pdocker 
+PersistentDispatcherMultipleConsumersPersistentDispatcherMultipleConsumers
```

Comment:
Please remove

Reply:
Whoops. Not sure how that got there. Should be fixed now.